modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/services/rest/TestrayRun.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {APIResponse, TestrayBuild} from '../../graphql/queries';
+
+export type TestrayRun = {
+	build: TestrayBuild;
+	dateCreated: string;
+	dateModified: string;
+	description: string;
+	environmentHash: string;
+	externalReferenceCode: string;
+	externalReferencePK: string;
+	externalReferenceType: string;
+	id: number;
+	jenkinsJobKey: string;
+	name: string;
+	number: string;
+	status: string;
+};
+
+const runsResource = '/runs';
```

Comment:
Please, remove this line and put this value in this file [/Run.tsx](https://github.com/pt-liferay-solutions/liferay-portal/pull/1679/files#diff-3da1171b0efc0d6fd85e267123946f392a9b28171f571cb3623a47a2a2fd2f21) resource property directly.